fluent-plugin-events/lib/fluent/plugin/in_events.rb

```diff
@@ -138,22 +139,21 @@ def start_watcher_thread
       end
 
       def create_config_map
-        @resource_version = 0
-        @configmap.data = { "resource-version": "#{@resource_version}" }
+        @configmap.data = { "resource-version-#{resource_name}": "#{@resource_version}" }
         @client.public_send("create_config_map", @configmap).tap do |map|
           log.debug "Created config map: #{map}"
         end
       end
 
-      def update_config_map
+      def patch_config_map
         pull_resource_version
-        @configmap.data = { "resource-version": "#{@resource_version}"}
-        @client.public_send("update_config_map", @configmap).tap do |map|
-          log.debug "Updated config map: #{map}"
+        @client.public_send("patch_config_map", "fluentd-config-resource-version", {data: { "resource-version-#{resource_name}": "#{@resource_version}"}}, "sumologic").tap do |map|
```

Comment:
I just realized we're creating this config map in the `sumologic` namespace, can we make this a configurable parameter? Since some customers might want to deploy their collection setup in other namespaces

Reply:
good call, will change that